h/models/group.py

```diff
@@ -64,6 +64,20 @@ class Group(Base, mixins.Timestamps):
 
     description = sa.Column(sa.UnicodeText())
 
+    #: Enforce scope match for annotations in this group.
+    #: For groups with 1-n scopes, only allow annotations for target
+    #: documents whose URIs match one of the group's scopes.
+    #: When disabled, annotations should be allowed web-wide.
+    #: This setting has no effect if the group does not have any scopes.
+    #: Enforcement is the responsibility of services (i.e. the model
+    #: layer does not enforce scope compliance).
```

Comment:
I wonder if this explanation belongs more in the service or API code that actually uses `enforce_scope` to implement this logic? Once that code exists. And just a much shorter docstring here.